format/proto/types.proto

```diff
@@ -6,6 +6,7 @@ enum DataType {
     HOLE = 2;
     TRIMMED = 3;
     PROPOSAL = 4;
+    CHECKPOINT = 5;
```

Comment:
I don't think Checkpoint should be a data type. It should just be a field in metadata